ember-toucan-core/src/components/form/checkbox-field.hbs

```diff
@@ -13,9 +13,11 @@
           aria-invalid={{if @error "true"}}
           aria-describedby="{{if @error field.errorId}}"
           id={{field.id}}
+          name={{@name}}
+          value={{@option}}
           @isDisabled={{@isDisabled}}
           @isIndeterminate={{@isIndeterminate}}
-          @value={{@value}}
+          @value={{this.isChecked}}
```

Comment:
So:
1. `@value` determines if the checkbox is checked? https://github.com/CrowdStrike/ember-toucan-core/blob/main/ember-toucan-core/src/components/form/controls/checkbox.ts#L36
2. `value` (no `@`) determines the "option" in a group of options?

Reply:
Yup yup, more info over on https://github.com/CrowdStrike/ember-toucan-core/pull/81#discussion_r1140586395!